guides/common/modules/con_kickstart-workflow.adoc

```diff
@@ -14,4 +14,4 @@ When you run a {ProjectName} Kickstart script, the script performs the following
 . It runs user defined snippets.
 
 .Additional resources
-For more information about Kickstart, see {RHELDocsBaseURL}8/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}8_.
+For more information about Kickstart, see {RHELDocsBaseURL}9/html/automatically_installing_rhel/automated-installation-workflow_rhel-installer[Automated installation workflow] in _Automatically installing RHEL{nbsp}9_.
```

Comment:
This is about Clients! I think your update is great but does not match your PR. Technically, we could still list EL7/8/9 here. Bumping to 9 as latest major RHEL is :+1:

Reply:
My bad, let's move it to https://github.com/theforeman/foreman-documentation/pull/3601